autoPyTorch/evaluation/train_evaluator.py

```diff
@@ -172,10 +172,8 @@ def fit_predict_and_loss(self) -> None:
 
             status = StatusType.SUCCESS
 
-            self.logger.debug("In train evaluator fit_predict_and_loss, num_run: {} loss:{}".format(
-                self.num_run,
-                loss
-            ))
+            self.logger.debug("In train evaluator fit_predict_and_loss, num_run: {} loss:{},"
```

Comment:
can you also add the status here? This is useful when you have to wait for a long time to get the runhistory.